utils/SwiftIntTypes.py

```diff
@@ -90,16 +90,16 @@ def numeric_type_names_Macintosh_only():
 
 def all_integer_binary_operator_names():
     return ['<<', '>>', '&*', '&', '&+', '&-', '|', '^']
-    
```

Comment:
Just my coding preference, I guess.

There's places with it, and places without it *in the same file*. So by removing it, there is more code consistency.